lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -939,36 +1017,21 @@ def setUp(self):
         self.addCleanup(httpretty.disable)
         self.register_get_user_response(self.user)
         self.url = reverse("comment-detail", kwargs={"comment_id": "test_comment"})
-        cs_thread = make_minimal_cs_thread({
-            "id": "test_thread",
-            "course_id": unicode(self.course.id),
-        })
-        self.register_get_thread_response(cs_thread)
-        cs_comment = make_minimal_cs_comment({
-            "id": "test_comment",
-            "course_id": cs_thread["course_id"],
-            "thread_id": cs_thread["id"],
-            "username": self.user.username,
-            "user_id": str(self.user.id),
-            "created_at": "2015-06-03T00:00:00Z",
-            "updated_at": "2015-06-03T00:00:00Z",
-            "body": "Original body",
-        })
-        self.register_get_comment_response(cs_comment)
-        self.register_put_comment_response(cs_comment)
 
-    def test_basic(self):
-        request_data = {"raw_body": "Edited body"}
-        expected_response_data = {
+    def expected_response_data(self, overrides=None):
+        """
+        create expected response data from comment update endpoint
+        """
+        response_data = {
             "id": "test_comment",
             "thread_id": "test_thread",
             "parent_id": None,
             "author": self.user.username,
             "author_label": None,
-            "created_at": "2015-06-03T00:00:00Z",
-            "updated_at": "2015-06-03T00:00:00Z",
-            "raw_body": "Edited body",
-            "rendered_body": "<p>Edited body</p>",
+            "created_at": "1970-01-01T00:00:00Z",
```

Comment:
If we want to test specific date propagated correctly then I would prefer to revert back this change and not use default date. It seems redundant to create a separate test for it.